developer_docs/how_to_create_a_plugin.asciidoc

```diff
@@ -2092,6 +2092,30 @@ Similarly for mutations:
 
 where `::Mutations::Ducks::Delete` is your delete mutation class inheriting from `::Mutations::DeleteMutation`.
 
+[[adding-subscribers]]
+=== Adding subscribers
+
+_Requires Foreman 1.25 or higher, set requires_foreman '>= 1.25' in
+engine.rb_
+
+You can consume events from Foreman core by registering subscribers. Subscriber is a class that inherits `::BaseSubscriber`. It is recommended to store subscribers under the `/app/subscribers/my_plugin/` directory. If you have your subscriber defined register it in the plugin. Example of your engine.rb:
+
+[source, ruby]
+....
+Foreman::Plugin.register :my_plugin do
+  # other code here
+  subscribe :my_event, MyPlugin::MySubscriber
+end
+....
+
+where `:my_event` is the name of the event you want to subscribe to. You may also subscribe to multiple events at once by using a regular expression, e.g. to subscribe to all events whose name starts with `foreman.` use:
+
+[source, ruby]
+....
+  subscribe /^foreman./, MyPlugin::MySubscriber
+....
+
+You can find all observable events by calling `Foreman::EventSubscribers.all_observable_events` in the Rails console. Keep in mind that this list only contains events that were set by the `set_hook` method.
```

Comment:
This is giving me without the "foreman." prefix, so I guess it's wrong. Need to test this.